test/checkpoint/test_checkpoint.c

```diff
@@ -298,18 +296,18 @@ wt_connect(const char *config_open)
           "\"%s\",file_manager=(close_handle_minimum=1,close_idle_time=1,close_scan_interval=1),"
           "log=(enabled),cache_size=1GB, eviction_dirty_trigger=%i, "
           "eviction_dirty_target=%i,%s%s%s%s",
-          progname, fast_eviction_trigger ? 5 : 20, fast_eviction_target ? 1 : 5,
-          timing_stress_config, g.debug_mode ? DEBUG_MODE_CFG : "", config_open == NULL ? "" : ",",
+          progname, fast_eviction ? 5 : 20, fast_eviction ? 1 : 5, timing_stress_config,
+          g.debug_mode ? DEBUG_MODE_CFG : "", config_open == NULL ? "" : ",",
           config_open == NULL ? "" : config_open));
-    else {
+    else
         testutil_check(__wt_snprintf(config, sizeof(config),
           "create,cache_cursors=false,statistics=(fast),statistics_log=(json,wait=1),log=(enabled),"
           "error_prefix=\"%s\",cache_size=1G, eviction_dirty_trigger=%i, "
           "eviction_dirty_target=%i,%s%s%s%s",
-          progname, fast_eviction_trigger ? 5 : 20, fast_eviction_target ? 1 : 5,
```

Comment:
Having eviction dirty target as 1 might be a bit scary. From my understanding the eviction update target is around 2 and is bigger. @kommiharibabu can you confirm?

Reply:
Yes, by default `eviction_update_target` is half of `eviction_dirty_target`. The `eviction_update_target` also gets changed automatically with `eviction_dirty_target` unless you set both of them separately.

Reply:
Right, should I be changing the eviction_dirty_target to 2 then? I noticed that in` conn_api.c`, the `__conn_close` function changes the `eviction_dirty_trigger = 1.0 `and `eviction_dirty_target =0.1` to encourage eviction threads to clear the dirty content, so not as sure what a suitable value would be here.